core/controllers/feedback.py

```diff
@@ -14,6 +14,8 @@
 
 """Controllers for the feedback thread page."""
 
+import feconf
```

Comment:
That's fine. But would it better for us to put `import ...` and `from ... import ...` in two separate blocks instead of mixing them up according to alphabet order? Usually `import ...` comes first.